### DISCUSSION TITLE
How to plot a side title in a pole figure? (not with "ColorBar" or "ylabel")

### DISCUSSION THREAD

**Discussion by kanerto** [mtex-toolbox/mtex#1491] [Ask Anything]
Does anyone know how to add a side title to a pole figure?
Adding a regular title overwrites the miller index.
When I plot a high number of grains I use colorful contours so I can add a title to the mtexColorBar,
but for a low number of grains, I'm only showing the original points for each orientation, so I can't use a color bar.
The function "Ylabel", used for cartesian graphs, doesn't work well for this kind of plotting.
Thanks

p.s. adding two illustrations -

<img width="321" alt="CPO_mO_split_below400_NAX2-stitch-0105 crc" src="https://user-images.githubusercontent.com/82959141/179403784-9f2f8dc2-a629-4437-ab77-e9907481c713.png">

<img width="331" alt="CPO_mO_split_above400_NAX2-stitch-0105 crc" src="https://user-images.githubusercontent.com/82959141/179403796-9878c93b-a026-495b-9ff1-eb5151dfdef0.png">

**Comment by zmichels** [comment]
Which part of the illustrations you included are meant to be an example of the “side title” you want to make?

**Reply by kanerto** [reply]
I want the bottom one to be without the color bar, just the side-title "1-point-per-grain" alone.
The color bar there is meaningless

**Comment by zmichels** [comment]
There may be some clever way to remove the colorbar portion afterwards, but I’m not sure and don’t have my computer to test.
It can definitely be done with text() though, which allows rotation… whereas I don’t think the annotation() approach allows rotation.

**Comment by filippeof** [comment]
hi,
you can play around with the coordinates, but something like this should work (see [text manual](https://mathworks.com/help/matlab/ref/text.html) for more options):
```matlab
f=newMtexFigure;
cs=crystalSymmetry("1")
plotPDF(orientation.rand(100,cs),Miller.rand(1,cs),'lower')

% make some space around the plot
setMTEXpref("outerPlotSpacing",30)
% plot text: text(x,y,str,...)
str='bottom'
t1=text(0.5, 0.001, str,'Units','normalized','VerticalAlignment','top','HorizontalAlignment','center','FontSize',10);
str='right'
t2=text(0.99, 0.5, str,'Units','normalized','VerticalAlignment','top','HorizontalAlignment','center','FontSize',10,'Rotation',90);
str='topRight'
t3=text(0.9, 0.99, str,'Units','normalized','VerticalAlignment','top','HorizontalAlignment','center','FontSize',10);
str='left'
t4=text(-0.05, 0.5, str,'Units','normalized','VerticalAlignment','top','HorizontalAlignment','center','FontSize',10,'Rotation',90);
```
cheers
Filippe
